### ISSUE TITLE
Image sizes do not match, even if embed specific formating is enabled

### ISSUE THREAD

**Issue by noblockhit** [brussell98/discord-markdown#35]
![image](https://user-images.githubusercontent.com/67117651/173841291-55474d33-4a93-4374-b190-495fd766d46c.png)
![image](https://user-images.githubusercontent.com/67117651/173841377-5a1c8a85-b77e-425b-af9e-1ef649bfbc69.png)

**Comment by brussell98** [comment]
Yes, this library does not provide Discord styles. It is simply a markdown parser. You will need to add styles yourself, or use a library that provides full components.

**Comment by noblockhit** [comment]
Im insecure of the rules that i'd have to implement, can you please provide
the source where you got your rules from?

Am Do., 16. Juni 2022 um 17:17 Uhr schrieb Brussell <
***@***.***>:
